src/components/LayoutFooter/index.tsx

```diff
@@ -85,7 +85,12 @@ const LayoutFooter: React.FC<Required<ILayoutModifiable>> = ({ modifiers }) => (
               </Link>
             </li>
             <li className={styles.linkItem}>
-              <Link href="/doc/user-guide/privacy" className={styles.link}>
+              <Link href="/doc/user-guide/gdrive-privacy" className={styles.link}>
+                Privacy Policy
+              </Link>
+            </li>
+            <li className={styles.linkItem}>
+              <Link href="/doc/user-guide/dropbox-privacy" className={styles.link}>
                 Privacy Policy
```

Comment:
Just need to differentiate then with something like
```suggestion
              <Link href="/doc/user-guide/gdrive-privacy" className={styles.link}>
                Privacy Policy for Google Drive integration
              </Link>
            </li>
            <li className={styles.linkItem}>
              <Link href="/doc/user-guide/dropbox-privacy" className={styles.link}>
                Privacy Policy for Dropbox integration
```

Reply:
Depending on how that looks though. May be too long as I suggested them?